UserApi/UserApi/Helper/UserProfileHelper.cs

```diff
@@ -25,23 +25,31 @@ public class UserProfileHelper
             {"Civil Money Claims", AdGroup.MoneyClaims}
         };
 
+        private static readonly Dictionary<AdGroup, string> CaseTypeMappings = new Dictionary<AdGroup, string>
+        {
+            { AdGroup.MoneyClaims, "Civil Money Claims" },
+            { AdGroup.FinancialRemedy, "Financial Remedy" }
```

Comment:
Its it not Civil Finacial remedy ?

Reply:
Nope, only the money claims is civil:
![image](https://user-images.githubusercontent.com/8461739/57923086-37064d00-7899-11e9-8d0e-23f2eecd7924.png)

But good double checking!